addon/pods/components/frost-list-item/component.js

```diff
@@ -9,8 +10,12 @@ export default Ember.Component.extend({
     this.set('_frostList', this.nearestOfType(FrostList))
   }),
 
-  isSelected: Ember.computed.reads('model.isSelected'),
-
+  isSelected: Ember.computed('model.isSelected', function () {
+    let modelIsSelect = this.get('model.isSelected')
+    modelIsSelect ? $(this.get('element')).parent().addClass('is-selected')
+    : $(this.get('element')).parent().removeClass('is-selected')
```

Comment:
I need to be able to bind the class to the parent. The parent is an [ember component](https://github.com/runspired/smoke-and-mirrors/blob/develop/addon/components/vertical-item.js). I can extend it to do the binding similar to [this](http://stackoverflow.com/a/18369777/3771965)